src/docs/truffle/reference/truffle-commands.md

```diff
@@ -144,13 +135,11 @@ If you want an interactive console but want to use an existing blockchain, use `
 
 See the [Using the console](/docs/getting_started/console) section for more details.
 
-
 Option:
 
-* `--log`: Start/Connect to a Truffle develop session and log all RPC activity.
-See the [Log RPC Activity](docs/getting_started/console#log-rpc-activity)
```

Comment:
Same as above.

Reply:
Oh, good catch, I'm going to replace it with this one: https://trufflesuite.com/docs/truffle/getting-started/using-truffle-develop-and-the-console.html